### PR TITLE
mtr: use GitHub release archive (via secure url)

### PR DESCRIPTION
**Pull request by vszakats**
* align build dependencies of stable with head

### CODE REVIEW

File: Library/Formula/mtr.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Mtr < Formula
   desc "'traceroute' and 'ping' in a single tool"
   homepage "https://www.bitwizard.nl/mtr/"
-  url "ftp://ftp.bitwizard.nl/mtr/mtr-0.86.tar.gz"
+  url "https://github.com/traviscross/mtr/archive/v0.86.tar.gz"
   sha256 "c5d948920b641cc35f8b380fc356ddfe07cce6a9c6474afe242fc58113f28c06"
```
Comment (bfontaine) [inline Library/Formula/mtr.rb]: It doesn’t seem to have the same checksum:
```
==> Downloading https://github.com/traviscross/mtr/archive/v0.86.tar.gz
==> Downloading from https://codeload.github.com/traviscross/mtr/tar.gz/v0.86
Warning: Formula reports different SHA256: c5d948920b641cc35f8b380fc356ddfe07cce6a9c6474afe242fc58113f28c06
Downloaded to: /Library/Caches/Homebrew/mtr-0.86.tar.gz
SHA1: 3fa90700b70b15f3f34babdb4d9e3dc3f29d36d7
SHA256: 7912f049f9506748913e2866068b7f95b11a4e0a855322120b456c46ac9eb763
```

File: Library/Formula/mtr.rb [hunk 2]
```diff
@@ -12,11 +13,7 @@ class Mtr < Formula
     sha1 "be91d5c1ad604d190ef1e1d56842592b816197bf" => :mountain_lion
   end
 
-  head do
-    url "https://github.com/traviscross/mtr.git"
-    depends_on "automake" => :build
-  end
-
+  depends_on "automake" => :build
```
Comment (bfontaine) [inline Library/Formula/mtr.rb]: The `system "./bootstrap.sh"` below is run only if `build.head?`, should it run unconditionally then?

File: Library/Formula/mtr.rb (hunk 1 above)
Comment (vszakats) [inline Library/Formula/mtr.rb]: Oops, sorry.

File: Library/Formula/mtr.rb (hunk 2 above)
Comment (vszakats) [inline Library/Formula/mtr.rb]: Good point, thanks! Corrected.

**Comment by rakkesh** [comment]
@vszakats while you are at it, could you add the patch suggested in issue https://github.com/Homebrew/homebrew/issues/43862#issuecomment-139821026?

**Comment by vszakats** [comment]
@rakkesh Added it in a second commit.

File: Library/Formula/mtr.rb [hunk 3]
```diff
@@ -1,8 +1,17 @@
...
   homepage "https://www.bitwizard.nl/mtr/"
-  url "ftp://ftp.bitwizard.nl/mtr/mtr-0.86.tar.gz"
-  sha256 "c5d948920b641cc35f8b380fc356ddfe07cce6a9c6474afe242fc58113f28c06"
+  head "https://github.com/traviscross/mtr.git"
+
+  stable do
+    url "https://github.com/traviscross/mtr/archive/v0.86.tar.gz"
+    sha256 "7912f049f9506748913e2866068b7f95b11a4e0a855322120b456c46ac9eb763"
+
+    patch do
```
Comment (mikemcquaid) [inline Library/Formula/mtr.rb]: Can you please add a explanation of why the patch is needed in a comment in the formula file. Thanks!

Comment (vszakats) [inline Library/Formula/mtr.rb]: Okay, done that. I hope the text is fine, if not, I'm open to update it.

Comment (mikemcquaid) [inline Library/Formula/mtr.rb]: It's great, thanks.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/98710d7c9b75a08229f385c213aef92d1c63f179 & https://github.com/Homebrew/homebrew/commit/290a8d29d8c310cfb76fd7489455aa8e5dd54f2c. Thanks @vszakats! :smiley_cat:

**Pull request closed without merging**